zwave_js_server/client.py

```diff
@@ -71,6 +71,10 @@ def __init__(self, ws_server_url: str, aiohttp_session: ClientSession):
         self._result_futures: Dict[str, asyncio.Future] = {}
         self._loop = asyncio.get_running_loop()
 
+    def __repr__(self) -> str:
+        """Return the representation."""
+        return f"{type(self).__name__}(ws_server_url={self.ws_server_url})"
```

Comment:
Should we wrap it like this `<Client(url)>` ?

Reply:
I normally try to create a string that can be passed to `eval` to create the instance again. Here we don't include all the parameters, so that wouldn't really work, but I'm still using that format.